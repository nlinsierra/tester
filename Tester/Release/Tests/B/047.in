7H 6H AH JH AD 5D 9H 4C 6C 9D 4S 5S 9S 6D JD 2H
