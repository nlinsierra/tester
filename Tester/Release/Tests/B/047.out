36
2C 3C 5C 7C 8C 9C TC JC QC KC AC 2D 3D 4D 7D 8D TD QD KD 3H 4H 5H 8H TH QH KH 2S 3S 6S 7S 8S TS JS QS KS AS
